dispatch/static/manager/src/js/reducers/AuthReducer.js

```diff
@@ -38,4 +39,12 @@ reducer.handle(fulfilled(types.AUTH.DELETE_TOKEN), () => {
   return initialState
 })
 
+reducer.handle(fulfilled(types.AUTH.VERIFY_TOKEN), (state) => {
+  return R.merge(state, { validToken: true })
+})
+
+reducer.handle(rejected(types.AUTH.VERIFY_TOKEN), (state) => {
+  return R.merge(state, { validToken: false })
```

Comment:
I can verify that the rejected handler runs when DNS lookup fails, so I can only assume it also does for timeouts. (for which I can't find a good way to simulate)